tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -1831,13 +1835,34 @@ final case class TestingServer(
       .toMap
       .updated("root", "root")
 
+    def label(uri: String, default: String): String =
+      labelsMap.get(uri) match {
+        /* fallback for JDK sources since we seem to have issues on CI
```

Comment:
I had to work around an issue in ubuntu CI jobs. We somehow had two different src.zip for Java in two different java homes. I wasn't able to reproduce or figure it out at any point and I think it would be waste of time to fix it since it only influences tests.

Reply:
Ok, maybe I will try one more thing.